src/main/java/org/launchcode/water_garden_tour/controllers/GardenController.java

```diff
@@ -158,7 +183,6 @@ public String updateGarden(Model model,
         return "gardens/admin-list";
     }
 
```

Comment:
I'm excited to see this!
Right now I'm getting an error on line 173 (`gardenToUpdate.setImage(image);`) when trying to compile: 
`error: incompatible types: String cannot be converted to byte[]`

Reply:
You're too quick for me!  I was still in the process of fixing/adjusting to handle the new image type and uploads with Ryan's changes.  Should work now.  Thanks!